webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -258,13 +258,20 @@ const NavigationSideBar: React.FC = () => {
             <section className={css.launch}>
               <div className={css.launchBlock}>
                 <div className={css.launchButton}>
-                  <Button block type="ghost" onClick={() => openJupyterLabModal()}>
+                  <Button
+                    block
+                    disabled={!canCreateNSC}
+                    type="ghost"
```

Comment:
Console says `Warning: [antd: Button] `link` or `text` button can't be a `ghost` button.`